base.cfg

```diff
@@ -99,6 +99,7 @@ mode = 755
 recipe = plone.recipe.codeanalysis
 directory = ${buildout:directory}/src
 flake8-exclude=bootstrap.py,bootstrap-buildout.py,docs,bin,*.egg,setup.py,overrides
+flake8-max-line-length = 88
```

Comment:
Black has a soft line length of 88, but lines may be longer (but iirc respect the 120 max). So this may produce false reports.

Reply:
Not sure why this was present in this PR. Anyways, we set the line length in master to 200 to not interfere with black...